spatial-backdrop/README.md

```diff
@@ -0,0 +1,127 @@
+# Spatial Backdrop Explainer
+
+The purpose of this document is to explain a new type of Link Tag for adding Immersive Backdrops from 3D models to Web Pages. The primary target for this is for standalone head mounted immersive displays but it should have application on other platforms too.
```

Comment:
It's not called a "Link Tag". It's called a link type. See https://html.spec.whatwg.org/#linkTypes